go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/jhump/protoreflect
 
 require (
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b // indirect
```

Comment:
I thought `indirect` dependencies were created when you run tools in this folder -- so they aren't actually dependencies of this module. Is that right? What did you run that added this?

Maybe you could try removing this line and doing `rm go.sum && go test ./...`, but _nothing else_. That way it only pulls in actual dependencies.

Reply:
You are right, it’s not a direct dependency of protoreflect, but it’s pulled in by `go mod tidy` as a dependency of `google.golang.org/grpc`. I’ve run the `go1.12 mod tidy` and it does the same thing, so I believe it was intentionally omitted from `go.mod`.

Reply:
Since `google.golang.org/grpc/grpclog/glogger` is not used directly in this module, it’s OK to omit the dependency from `go.mod`. See bcee28a8875a05bb488e1147b6d1d22315b15e52.